appveyor.yml

```diff
@@ -49,8 +100,8 @@ init:
   - cmd: cd "C:\projects\"
   - cmd: md pthreads-win32
   - cmd: cd "C:\projects\"
-  - appveyor DownloadFile https://github.com/tango-controls/Pthread_WIN32/releases/download/2.9.1/pthreads-win32-2.9.1_%ARCH%.zip
-  - cmd: 7z -y x pthreads-win32-2.9.1_%ARCH%.zip -oC:\projects\pthreads-win32\
+  - appveyor DownloadFile https://github.com/mnabywan/Pthread_WIN32/releases/download/%PTHREAD_VERSION%/pthreads-win32-%PTHREAD_VERSION%_%ARCH%.zip
```

Comment:
I did "unofficial" release on my fork  of Pthread (2.9.2) to check if it works:
https://github.com/mnabywan/Pthread_WIN32/releases/tag/2.9.2